solver/src/main.rs

```diff
@@ -117,6 +118,15 @@ struct Arguments {
         parse(try_from_str = shared::arguments::wei_from_base_unit)
     )]
     min_order_size_one_inch: U256,
+
+    /// The list of tokens our settlement contract is willing to buy when settling trades
+    /// without external liquidity
+    #[structopt(
+        long,
+        env = "MARKET_MAKEABLE_TOKEN_LIST",
+        default_value = "https://tokens.coingecko.com/uniswap/all.json"
+    )]
+    market_makable_token_list: String,
```

Comment:
I suppose there are plenty of reasons this should be different from the base tokens we already use. 

I like the default value, but it seems like quite a vast list of tokens I've never heard of. In fact this list even includes NAWA (the token that was the reason for introducing this).